filebeat/module/elasticsearch/audit/ingest/pipeline-json.json

```diff
@@ -66,28 +74,28 @@
         },
         {
             "set": {
-                "if": "ctx.elasticsearch.audit?.url?.path != null && ctx.elasticsearch.audit?.url?.query == null",
                 "field": "url.original",
+                "if": "ctx.elasticsearch.audit?.url?.path != null \u0026\u0026 ctx.elasticsearch.audit?.url?.query == null",
```

Comment:
I wonder if `\u0026\u0026` is still correct. HTML escaping should be disabled if it's cause by it.

Reply:
yeah, we should leave this in a human-friendly format :slightly_smiling_face: